### ISSUE TITLE
n-link CLI - thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: LibUsb', src/cli.rs:98:16

### ISSUE THREAD

**Issue by Frityet** [lights0123/n-link#17]
OS: MacOS 10.15.7 "Catalina"
Calc: Nspire CX-2 CAS- Version 5.2.0.771

When trying to upload a program I made, I get this error. Using the GUI makes the upload "frozen", forcing a program restart

**Comment by Frityet** [comment]
<img width="859" alt="Screen Shot 2021-11-29 at 8 06 52 AM" src="https://user-images.githubusercontent.com/25426233/143902419-f8f7d53d-60e4-4e84-95c5-205a7dd5a097.png">

**Comment by Frityet** [comment]
I do also have an up-to-date version of `libusb` (1.0.24) installed with homebrew

**Comment by Frityet** [comment]
This seems to be fixed on the development branch, as building the repo and running it works perfectly

**Comment by Frityet** [comment]
<img width="781" alt="Screen Shot 2021-11-29 at 11 52 53 AM" src="https://user-images.githubusercontent.com/25426233/143933494-83928632-5304-4ac4-a97d-6141a57af808.png">
Reopening because a new error appeared

**Comment by lights0123** [comment]
Yep, the latest version formats errors differently, although I'm considering reverting that.
